packages/compass-workspaces/src/stores/workspaces.ts

```diff
@@ -3,28 +3,9 @@ import type { ThunkAction } from 'redux-thunk';
 import { ObjectId } from 'bson';
 import AppRegistry from 'hadron-app-registry';
 import toNS from 'mongodb-ns';
-import type { WorkspacesServices } from '..';
-import type { MyQueriesWorkspace } from '@mongodb-js/compass-saved-aggregations-queries';
-import type { ServerStatsWorkspace } from '@mongodb-js/compass-serverstats';
-import type {
-  DatabasesWorkspace,
-  CollectionsWorkspace,
-} from '@mongodb-js/compass-databases-collections';
-import type { CollectionWorkspace } from '@mongodb-js/compass-collection';
```

Comment:
Reversing the dependency here, now workspaces plugin is the source of truth for workspaces interface, to avoid circular dependency